compat/mingw.h

```diff
@@ -501,6 +501,7 @@ static inline void convert_slashes(char *path)
 #define find_last_dir_sep mingw_find_last_dir_sep
 int mingw_offset_1st_component(const char *path);
 #define offset_1st_component mingw_offset_1st_component
+#define CAN_UNLINK_MOUNT_POINTS 1
```

Comment:
This definition belongs into 81e4da1062beec67d2e1fbcbec3c2da3d83a0e1d; Could I ask you to move this hunk into that patch?